### PR TITLE
New README and python asset samples

### PR DESCRIPTION
**Pull request by rick-kenna**
New samples for upcoming blog on setting an asset inactive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+For more samples, please look at our [All_Samples](https://github.com/KennaSecurity/All_Samples)
+repository.  There you will find many samples, mostly in Ruby.
+
+For additional information about how you can use the Kenna API (employed in the sample scripts)
+to enhance/supplement your Kenna implementation, contact
+[Kenna Customer Success](mailto:customersuccess@kennascurity.com).
+
+All the code samples in this GitHub repository are offered “as is” and include no warranty of any kind.
+Use them at your own risk. In no event will Kenna be liable to end user or any other party for damages
+of any kind arising from the use of these samples.
```
Comment (sfgeorge) [inline README.md:19]: Good call-out here.  Can you copy the same MIT License that we have on toolkit?

KennaSecurity/toolkit@5bd1af20d9faeea03360a05672503afead7b13f7

File: python/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Contains Kenna Security code samples that are relevant to Kenna Security blogs. The directories
+reflect the sections in the [API documentation](https://apidocs.kennasecurity.com/reference).
+
+The examples use the environment variable `API_KEY`. Please make sure you follow the
+[authentication directions](https://apidocs.kennasecurity.com/reference#authentication) 
+ and export the API_KEY in your shell environment.
+
+The python samples use the [requests library](https://requests.readthedocs.io/en/master/) to communincate
+to Kenna Security servers over HTTP.
```
Comment (sfgeorge) [inline python/README.md:11]: Awesome overall docs 👍 

minor correction: spelling of communicate

File: python/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+These samples were tested using Python 3.8.7.
```
Comment (sfgeorge) [inline python/README.md:13]: Can we slap a `.python-version` file in the directory to be explicit?

File: python/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Kenna Security Python Blog Samples
+
+Contains Kenna Security code samples that are relevant to Kenna Security blogs. The directories
+reflect the sections in the [API documentation](https://apidocs.kennasecurity.com/reference).
+
+The examples use the environment variable `API_KEY`. Please make sure you follow the
+[authentication directions](https://apidocs.kennasecurity.com/reference#authentication) 
+ and export the API_KEY in your shell environment.
+
+The python samples use the [requests library](https://requests.readthedocs.io/en/master/) to communincate
```
Comment (sfgeorge) [inline python/README.md:10]: Be it via `requirements.txt`, `Pipfile` or similar, can we add a manifest with the required Python packages?

Comment (rick-kenna) [reply python/README.md:10]: Thank-you for the reminder.  I too old school and forgot about requirements.txt.

File: python/README.md (hunk 2 above)
Comment (rick-kenna) [reply python/README.md:11]: Fixed.

File: README.md (hunk 1 above)
Comment (rick-kenna) [reply README.md:19]: Added LICENSE.md.

File: python/README.md (hunk 3 above)
Comment (rick-kenna) [reply python/README.md:13]: Done.  What is your opinion about having it .gitignore?

Comment (rick-kenna) [reply python/README.md:13]: Since @sfgeorge hasn't responded, I'm going to not add .python-version to .gitignore.  It can be done later.

Comment (sfgeorge) [reply python/README.md:13]: I'm open to your thoughts on this one.  And if you'd lean towards git-ignoring `.python-version` I can get behind that.

`.python-version` obviously should not be checked-in library, such as an SDK.  But for an application (which this feels more like to me), it's okay to be explicit about a python/language version.

I use the same distinction for ...`.lock` files...  In a library: no, In an application: possibly yes.

**Pull request merged**